pkg/apis/scorecard/v1alpha3/configuration_types.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha3
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ConfigurationKind is the default scorecard componentconfig kind.
+const ConfigurationKind = "ScorecardConfiguration"
+
+// Configuration represents the set of test configurations which scorecard would run.
+type Configuration struct {
+	metav1.TypeMeta `json:",inline" yaml:",inline"`
+
+	// Do not use metav1.ObjectMeta because this "object" should not be treated as an actual object.
+	Metadata struct {
+		Name string `json:"name,omitempty" yaml:"name,omitempty"`
+	} `json:"metadata,omitempty" yaml:"metadata,omitempty"`
```

Comment:
Is this needed to make `kustomize` happy? If so, we should probably document that in a comment.

It also gives me pause that we're misusing kustomize, which maybe is meant to be used with applyable objects. Thoughts?

Reply:
It is, but I have a feeling that component configs are going to be used often enough in the future that kustomize will support them eventually.